Dockerfile

```diff
@@ -1,8 +1,143 @@
 # Build the manager binary
-FROM --platform=$BUILDPLATFORM golang:1.20.1 as builder
+FROM --platform=$BUILDPLATFORM debian:bookworm-slim as builder
```

Comment:
Why do we setup golang entirely on our own here? The default `golang:1.20.1` points to a version of Debian bullseye, which is the latest version of debian. bookworm is still in testing phase, isn't it?

Reply:
`libcephfs-dev`  `librbd-dev` `librados-dev` is not present in `bullseye` therefore the other option would be to build this three libs from scratch. Once the new `go image` uses `bookworm` the manual go building stuff will be removed and the rest stays the same.